coupon-service/src/main/resources/application-batch-dev.yml

```
@@ -0,0 +1,20 @@
+    job:
+      enabled: false # 자동 job 실행 비활성화
+    jdbc:
+      initialize-schema: always
+
+# 웹 서버를 아예 띄우지 않음
+server:
+  port: -1
+
+management:
+  tracing:
+    enabled: false
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)